README.md

```diff
@@ -11,7 +11,14 @@ Various Java client libraries for utilization of APIs in <a href="https://adscor
  <li> JS: https://github.com/variably/adscore-node </li>
 </ul>
 
-<h3>How to use</h3>
+<h4>How to pull sample submodule</h4>
+Execute:
+```
```

Comment:
I think I would move this section into 'how to use' and provide this as a example really ;)

Reply:
Done!

Reply:
lets put also a note about sample here:
```
Following are few quick examples of how to use verifier, first import the entry point for library:
```

as I think it would be good to let know user that there is a sampel there ;)

Reply:
Okay, added what you've suggested + info about samples readme, where info on how to execute samples is stored.